Braze Demo Match Game Content Extension/ViewController/NotificationViewController.swift

```diff
@@ -0,0 +1,185 @@
+import UIKit
+import UserNotifications
+import UserNotificationsUI
+
+class NotificationViewController: UIViewController, UNNotificationContentExtension {
```

Comment:
One possibility, since this is meant to be educational, would be to mark methods and functions that most implementors would expect to have to build for their own notifications - so that they can quickly sort through what's specific to this app vs what's general to content extensions

Reply:
I like that idea. Code-wise, I like how it's organized with `public` and private` extensions but I can create separate extensions to further organize it between important call-outs (saving custom event data) and just arbitrary match game code.